schema.yml

```diff
@@ -80,9 +80,7 @@ mapping:
 
           Environment variables with only a key are resolved to their values on the machine Compose is running on, which
```

Comment:
The description here says "You can use an array"  but that's only partially true now.